source/Handlebars/HtmlEncoder.cs

```diff
@@ -34,14 +61,16 @@ public string Encode(string text)
                         if (text[i] > 159)
                         {
                             sb.Append("&#");
-                            sb.Append(((int) text[i]).ToString(CultureInfo.InvariantCulture));
+                            sb.Append(((int)text[i]).ToString(CultureInfo.InvariantCulture));
                             sb.Append(";");
                         }
                         else
```

Comment:
@304NotModified I guess I see the braces existed like this before your edits. So maybe this would be an appropriate time to fix?